ocpp/charge_point.py

```diff
@@ -54,6 +54,8 @@ def snake_to_camel_case(data):
         for key, value in data.items():
             key = key.replace("soc", "SoC")
             key = key.replace("_v2x", "V2X")
+            key = key.replace("_url", "URL")
```

Comment:
The issue with this approach is another field, as I mentioned in my other PR. The ocppCsmsUrl field in a SetNetworkProfileRequest will fail due to this change. I would suggest adding test cases for any usage of URL in the spec.

Reply:
I believe these are all attributes that include `url`:

```
$ grep -i "url" ocpp/v201/{call,call_result,datatypes}.py
ocpp/v201/datatypes.py:    ocpp_csms_url: str
ocpp/v201/datatypes.py:    responder_url: str
```